WeakAurasOptions/OptionsFrames/OptionsFrame.lua

```diff
@@ -661,21 +661,21 @@ function WeakAuras.CreateFrame()
     self:FillOptions(displayOptions[tempGroup.id]);
   end
 
-  frame.ClearPicks = function(self, except)
+  frame.ClearPicks = function(self, noHide)
```

Comment:
There's a caller that calls ClearPicks with a aura name: 
WeakAuras.ClearPicks(data.id);

Reply:
"except" is not used by the function

Reply:
or i'm blind

Reply:
That doesn't mean you should fix the call that passes a parameter!